### DISCUSSION TITLE
📰 What's New at Battlesnake in 2023  🎉

### DISCUSSION THREAD

**Discussion by andrewdmaclean** [BattlesnakeOfficial/feedback#222] [Announcements & News]
We're thrilled share the exciting changes we’ve made to Battlesnake including our website’s new look, new player ranking system, and a new way to compete! 🐍

![image](https://user-images.githubusercontent.com/26526271/212771134-438465e5-475c-4eba-b7df-3227b0b32b93.png)

### Learn more at [docs.battlesnake.com/blog/2023/01/16/whats-new-at-battlesnake-in-2023](https://docs.battlesnake.com/blog/2023/01/16/whats-new-at-battlesnake-in-2023) 👀

**Comment by KevinBanana** [comment]
updated link: [https://docs.battlesnake.com/blog/2023/01/16/whats-new-at-battlesnake-in-2023](https://docs.battlesnake.com/blog/2023/01/16/whats-new-at-battlesnake-in-2023)

**Reply by coreyja** [reply]
Thanks @KevinBanana, I updated the link in the original message too!
